inspirehep/config.py

```diff
@@ -596,3 +598,16 @@
 }
 """Updates default properties that should be added to the Holding Pen index
 mappings."""
+
+CACHED_VIEWS = {
+    'single_entry': {
```

Comment:
Ah, I see. Shall we rename it? :dancer:  Say, `record_reference`  (reference/citation is the same?) or `record_oneline`. I am thinking it could be good to also rename `brief` to `record_brief` so that the full cache name becomes more clear `next::_template_fragment_cache_brief_1227953` vs `next::_template_fragment_cache_record_brief_1227953